gpt_index/indices/knowledge_graph/base.py

```diff
@@ -121,6 +123,20 @@ def _build_index_from_documents(self, documents: Sequence[BaseDocument]) -> KG:
                 logging.debug(f"> Extracted triplets: {triplets}")
                 for triplet in triplets:
                     index_struct.upsert_triplet(triplet, n)
+
+                if self.include_embeddings:
+                    for i, triplet in enumerate(triplets):
+                        self._embed_model.queue_text_for_embeddding(
```

Comment:
these parentheses aren't the prettiest but i'll fix it later

Reply:
ah, I think `black` re-formatted that. I can fix that as well, that's pretty nasty haha